htdocs/admin/facture.php

```diff
@@ -739,6 +765,28 @@
 print "<br>";
 print load_fiche_titre($langs->trans("OtherOptions"), '', '');
 
+if ($conf->global->MAIN_FEATURES_LEVEL >= 2) {
+
+// Editing global variables not related to a specific theme
+$constantes=array(
+    'FAC_FORCE_DATE_VALIDATION'=>array('type'=>'yesno', 'label'=>$langs->trans('FAC_FORCE_DATE_VALIDATION')),
+    'INVOICE_EMAIL_TEMPLATE_INVOICE_VALIDATION'=>'emailtemplate:facture_send',
+    'INVOICE_EMAIL_TEMPLATE_INVOICE_PAID'=>'emailtemplate:member',
+    'INVOICE_EMAIL_TEMPLATE_INVOICE_DISPUTE'=>'emailtemplate:member', //dispute on online payment
+    'INVOICE_EMAIL_TEMPLATE_INVOICE_CANCELATION'=>'emailtemplate:member',
+    'INVOICE_FREE_TEXT'=>'html',
+    'FACTURE_DRAFT_WATERMARK'=>'string'
+);
+
+$substitutionarray=pdf_getSubstitutionArray($langs, null, null, 2);
+$substitutionarray['__(AnyTranslationKey)__']=$langs->trans("Translation");
+$helptext = '<i>'.$langs->trans("AvailableVariables").':<br>';
+foreach($substitutionarray as $key => $val)	$helptext.=$key.'<br>';
+$helptext.='</i>';
+
+form_constantes($constantes, 0, $helptext);
```

Comment:
New use of form_constantes must use the value 2 for the second parameter.
Does it works with 2 instead of 0 ?